cmd/tofu/oci_distribution.go

```diff
@@ -114,6 +117,44 @@ var _ ociauthconfig.CredentialsLookupEnvironment = ociCredentialsLookupEnv{}
 
 // QueryDockerCredentialHelper implements ociauthconfig.CredentialsLookupEnvironment.
 func (o ociCredentialsLookupEnv) QueryDockerCredentialHelper(ctx context.Context, helperName string, serverURL string) (ociauthconfig.DockerCredentialHelperGetResult, error) {
-	// TODO: Implement this
-	return ociauthconfig.DockerCredentialHelperGetResult{}, fmt.Errorf("support for Docker-style credential helpers is not yet available")
+	// (just because this type name is very long to keep repeating in full)
+	type Result = ociauthconfig.DockerCredentialHelperGetResult
```

Comment:
Nit, feel free to ignore:
I wonder what your thoughts are on `var result ociauthconfig.DockerCredentialHelperGetResult` as an alternative.

Reply:
Funnily enough I did write it that way at first, but I switched it because I liked being able to write a composite literal directly in the `return` statement at the end.

I'm happy to change it back if this seems distasteful or too odd for your tastes. I would agree that it's an unusual technique.

Reply:
I think it's reasonable and don't need you to change it.  I do kind of like that the struct initialization happens further down which might be easier to grep for?